lib/internal/Magento/Framework/Stdlib/DateTime/TimezoneInterface.php

```diff
@@ -139,10 +141,29 @@ public function formatDateTime(
     );
 
     /**
+     * Convert date from config timezone to Utc.
+     *
+     * If pass \DateTime object as argument be sure that timezone is the same with config timezone
+     *
      * @param string|\DateTimeInterface $date
      * @param string $format
+     * @throws LocalizedException
      * @return string
-     * @since 100.1.0
+     * @deprecated
      */
     public function convertConfigTimeToUtc($date, $format = 'Y-m-d H:i:s');
+
+    /**
+     * Convert date from config timezone to Utc.
+     *
+     * If pass \DateTime object as argument be sure that timezone is the same with config timezone
+     *
+     * @param string|\DateTimeInterface $date
+     * @param string $format
+     * @param string $pattern
+     * @throws LocalizedException
+     * @return string
+     * @deprecated
+     */
+    public function convertConfigTimeToUtcWithPattern($date, $format = 'Y-m-d H:i:s', $pattern = 'Y-m-d H:i:s');
```

Comment:
Adding a public method into interface marked as API is backward incompatible and therefore is forbidden (please look BIC guide https://devdocs.magento.com/guides/v2.3/contributor-guide/backward-compatible-development/). Please reimplement it. You can use the possible approach how to do that in the mentioned guide.